### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by geoknee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,89 @@
-# Sample Hardhat Project
+<h1 align="center">
+<div><img src="./SCBridge-Wallet.png"><br>
+SCBridge-Wallet
+</h1>
+<h3 align="center">
+An L2 state channel bridge contract which is also an L1 smart contract wallet.
```
Comment (NiloCK) [inline README.md:6]: Maybe:

```suggestion
An L2 state channel bridge contract which is also an EIP-4337 compliant smart contract wallet.
```

File: README.md [hunk 2]
```diff
@@ -1,13 +1,89 @@
...
+group Multihop L2 payment from Alice to Bob via Irene
+Alice->Bob: request an invoice
+Bob->Alice: hash
+Alice->Irene: add HTLC(hash,timeout,amount, sig, turnNum)
+Irene->Bob: add HTLC
+Bob->Irene:  preimage + updated signed state
+Irene->Alice: preimage + updated signed state
+Irene->Bob: countersigned updated state
+Alice->Irene: countersigned updated state
+end
```
Comment (NiloCK) [inline README.md:88]: Maybe the l2 tx diagram should live inside the SCBridgeWallet box.

```suggestion

group Multihop L2 payment from Alice to Bob via Irene
Alice->Bob: request an invoice
Bob->Alice: hash
Alice->Irene: add HTLC(hash,timeout,amount, sig, turnNum)
Irene->Bob: add HTLC
Bob->Irene:  preimage + updated signed state
Irene->Alice: preimage + updated signed state
Irene->Bob: countersigned updated state
Alice->Irene: countersigned updated state
end

end
```

**Review by NiloCK — APPROVED** [review]
:+1:

File: README.md (hunk 2 above)
Comment (geoknee) [reply README.md:88]: I had it that way originally, but then I thought it applies equally well to both approaches I think? As in, it's the L1 behaviour that's substantially different between the two models?

**Pull request merged**